synapseclient/entity.py

```diff
@@ -240,6 +240,9 @@ def __init__(self, properties=None, annotations=None, local_state=None, parent=N
                 if 'annotations' in properties and isinstance(properties['annotations'], collections.abc.Mapping):
                     annotations.update(properties['annotations'])
                     del properties['annotations']
+                if 'items' in properties:
```

Comment:
Can you add a comment like:
```
#HACK: items is returned from dataset API but items() is a method in python dict
```

Reply:
ahem, it's not a hack. imo, it's a pretty smart solution!

Reply:
Sorry - didn't mean for that to be interpreted as not a smart solution!  It is indeed!